WebContent/jsp/common/allImports.jsp

```diff
@@ -0,0 +1,24 @@
+

```

Comment:
It's nice to add all the imports in one JSP and include that in index.jsp
so that if at all you want the same import many jsp, you can simply include this.